### PR TITLE
Byi override yaml update

### PR DESCRIPTION
**Pull request by bin3377**
update yaml files for:
- fix the changed name/job in metrics
- add cluster label
- remove cadvisor source since it's combined into kubelet

### CODE REVIEW

File: deploy/helm/overrides.yaml [hunk 1]
```diff
@@ -1,56 +1,137 @@
 prometheus:
   prometheusSpec:
+    externalLabels:
+      # Set this to a value to distinguish between different k8s clusters
+      cluser: kubernetes
```
Comment (frankreno) [inline deploy/helm/overrides.yaml]: missing the t in cluster

**Comment by bin3377** [comment]
@ggarg2906sumo this change should fix all missing time series. These things need to be noticed:

1. There is an incompatible between Prometheus-Operator chart and latest version of Kubernetes. So we need a workaround for install helm chart
 a) clone https://github.com/helm/charts/
 b) modify `values.yaml`: find `component: kube-controller-manager` and replace to `k8s-app: kube-controller-manager`; find `component: kube-scheduler` and replace to `k8s-app: kube-scheduler`
 c) install chart with modified values.yaml with running following command in <path-to-charts-repo>/stable:
`helm install --namespace sumologic --name prometheus-operator ./prometheus-operator -f prometheus-operator/values.yaml -f <path-to-sumologic-kubernetes-collection>/deploy/helm/overrides.yaml --no-crd-hook`

2. time series in cadvisor is combined into kubelet 

3. some metrics name changed in [node-exporter part](https://docs.google.com/document/d/1z0KFYygDMahOXsXrAYPOwmSkTmSDTOhUdVwOwxXusv0/edit#heading=h.yz65u4z4c7x1)

**Comment by frankreno** [comment]
why clone? why not set in helm overrides as it can override those value?

**Comment by frankreno** [comment]
I also think that due to the version change and use of many k8s versions in the wild, we should have an FAQ doc or something and just call this out specifically.

**Comment by bin3377** [comment]
I tried with make these changes into override.yaml, but it will not overwrite the original selector but add them as:
```
Name:              prometheus-operator-kube-controller-manager
...
Selector:          component=kube-controller-manager,k8s-app=kube-controller-manager
```

**Comment by frankreno** [comment]
Crap. I forgot about that bug in helm. Ok is what it is for the moment.

**Comment by bin3377** [comment]
@lei-sumo @samjsong

**Pull request merged**